ORMSolution/ORM/ORM/ORMEntity.cs

```diff
@@ -144,8 +141,48 @@ public ORMEntity(bool disableChangeTracking = false) : this()
         /// <returns>Returns the provided column value.</returns>
         public object this[string columnName]
         {
-            get { return GetType().GetProperty(columnName, PublicIgnoreCaseFlags | NonPublicFlags).GetValue(this); }
-            set  { GetType().GetProperty(columnName).SetValue(this, value); }
+            get
+            {
+                var propertyInfo = GetType().GetProperty(columnName, PublicIgnoreCaseFlags | NonPublicFlags);
+
+                if (propertyInfo == null)
+                {
+                    foreach (var property in GetType().GetProperties())
+                    {
+                        var columnAttribute = property.GetCustomAttributes(typeof(ORMColumnAttribute), false).FirstOrDefault() as ORMColumnAttribute;
+
+                        if (columnAttribute?.ColumnName == columnName)
+                        {
+                            return GetType().GetProperty(property.Name, PublicIgnoreCaseFlags | NonPublicFlags).GetValue(this);
+                        }
+                    }
+                }
```

Comment:
Kan dit niet naar een gescheiden functie? Deze code lijkt vrijwel duplicated met uitzondering van de getvalue(this)|setvalue(this,value).

Misschien dat we dit in de toekomst willen cachen per type, maar dat lijkt me voor dit PR overkill

Reply:
Uiteindelijk willen we een ReflectionHelper class maken toch? Dan kunnen we dit soort cases mooi maken.